pkg/pillar/zedcloud/send.go

```diff
@@ -221,31 +223,15 @@ func SendOnAllIntf(ctxWork context.Context, ctx *ZedCloudContext, url string, re
 	return nil, nil, senderStatus, err
 }
 
-// VerifyAllIntf : verify the state of connectivity for *all* uplink interfaces.
```

Comment:
This comment change for VerifyAllIntf (from newer to older) should not be part of this backported patch.

Reply:
Thanks. Reverted.